CHARTER.adoc

```diff
@@ -23,9 +23,10 @@ The TODO Group shall be composed of the General Members and a Steering Committee
  .. Composition:
  ... Steering Committee members shall be nominated or appointed for their expertise, contribution to the advancement of open source program management, and diversity of backgrounds and opinions.
  ... The Steering Committee shall be composed of seven (7) General Members, with five (5) elected and two (2) appointed.
- ... Committee member terms shall begin on January 1st of the year following election or appointment, and terminate on December 31st of the second year following election or appointment.
- ... In each calendar year, the group shall hold an election to fill elected positions which will be vacated at the end of that year. The election process shall include nominations and voting during the month of September.
- ... In each calendar year, the committee shall appoint one (1) member to fill the appointed position which will be vacated at the end of that year. The appointment shall be made during the month of October, by the current members of the commitee at that time.
+ ... Elected committee member terms shall begin on January 1st of the year following election, and terminate on December 31st of the second year following election.
+ ... Appointed committee member terms shall begin on the first day of the month after their appointment, and terminate on the last day of the month exactly 24 months after their appointment, unless the appointment is to replace a departed member as detailed below. Typically an appointed term will be February 1st for term start and January 31st two years later for term end.
```

Comment:
"Typically an appointed term will be February 1st for term start and January 31st two years later for term end."

This sentence sticks out to me a little, as it's language that serves as guidance, but isn't binding or prescriptive the way the rest of this section is. If it's included as an example, I think we should clarify the language a bit to say it's an example. If it's included to drive the SC to make a decision and not drag their feet, I think we should clarify the language in a way that sets it as a clear expectation of the SC. 

While I favor a clarification to this piece of language, I don't think it's enough of an issue to hold up the proposed changes.

Reply:
Fixed in the next commit.